src/isValidRequest.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+'use strict';
+
+const allowedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function isValidRequest(text, toCase) {
+  const statusText = {
+    errors: [],
+  };
```

Comment:
Is it necessary to wrap it in an object with 1 property?
```suggestion
  const errors = [];
```